app/models/application_record.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationRecord < ActiveRecord::Base
```

Comment:
I see that this is an addition of Rails 5.1, learn a new thing every day! https://blog.bigbinary.com/2015/12/28/application-record-in-rails-5.html
We should remove this for now though, since we are on Rails 5 currently and when we upgrade to 5.1, we'll want to also go through and change the inheritance of every model.